src/package/cppwinrt/nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -130,77 +134,30 @@ Copyright (C) Microsoft Corporation. All rights reserved.
-            Returns="@(CppWinRTDynamicWinMDReferences)">
-        <!--To evaluate wildcard expansion of an item collection, must use intermediate property-->
-        <ItemGroup>
-            <_CppWinRTDynamicWinMDReferenceWildcard Remove="@(_CppWinRTDynamicWinMDReferenceWildcard)"/>
-            <_CppWinRTDynamicWinMDReferenceWildcard Include="@(CppWinRTDynamicProjectReferences->'%(RootDir)%(Directory)*.winmd')"/>
-            <_CppWinRTDynamicWinMDReferenceWildcard Condition="'%(_ResolvedNativeProjectReferencePaths.ProjectType)'!='StaticLibrary'"
-                Include="@(_ResolvedNativeProjectReferencePaths->WithMetadataValue('FileType','winmd')->'%(RootDir)%(Directory)*.winmd')"/>
-        </ItemGroup>
-        <PropertyGroup>
-            <_CppWinRTDynamicWinMDWildcard>@(_CppWinRTDynamicWinMDReferenceWildcard)</_CppWinRTDynamicWinMDWildcard>
-        </PropertyGroup>
-        <ItemGroup>
-            <_CppWinRTDynamicWinMDReferences Include="$(_CppWinRTDynamicWinMDWildcard)"/>
-            <_CppWinRTManagedWinMDReferences Include="@(_ResolvedNativeProjectReferencePaths->WithMetadataValue('FileType',''))" Condition= "$([System.String]::Copy('%(Identity)').EndsWith('.winmd',true,null))" />
-            <CppWinRTDynamicWinMDReferences Remove="@(CppWinRTDynamicWinMDReferences)"/>
-            <CppWinRTDynamicWinMDReferences Include="@(_CppWinRTDynamicWinMDReferences);@(_CppWinRTManagedWinMDReferences)">
-                <WinMDPath>%(Identity)</WinMDPath>
-            </CppWinRTDynamicWinMDReferences>
+            <!-- Get static library project references -->
+            <_CppWinRTStaticProjectReferences Remove="@(_CppWinRTStaticProjectReferences)"/>
+            <_CppWinRTStaticProjectReferences Include="@(_ResolvedProjectReferencePaths)"
+                Condition= "'%(_ResolvedProjectReferencePaths.ProjectType)'=='StaticLibrary' AND '%(_ResolvedProjectReferencePaths.WinMDFile)' == 'true'"/>
+            <CppWinRTStaticLibraryProjectWinMDReferences Remove="@(CppWinRTStaticLibraryProjectWinMDReferences)"/>
+            <CppWinRTStaticLibraryProjectWinMDReferences Include="@(_CppWinRTStaticProjectReferences)">
+                <WinMDPath>%(FullPath)</WinMDPath>
+            </CppWinRTStaticLibraryProjectWinMDReferences>
+            <!--Get dynamic library project references-->
+            <_CppWinRTDynamicProjectReferences Remove="@(_CppWinRTDynamicProjectReferences)"/>
+            <_CppWinRTDynamicProjectReferences Include="@(_ResolvedProjectReferencePaths)"
+                Condition= "'%(_ResolvedProjectReferencePaths.ProjectType)'!='StaticLibrary'  AND '%(_ResolvedProjectReferencePaths.WinMDFile)' == 'true'"/>
+            <CppWinRTDynamicLibraryProjectWinMDReferences Remove="@(CppWinRTDynamicLibraryProjectWinMDReferences)"/>
+            <CppWinRTDynamicLibraryProjectWinMDReferences Include="@(_CppWinRTDynamicProjectReferences)">
+                <WinMDPath>%(FullPath)</WinMDPath>
```

Comment:
without wildcard support, how is a dynamic project reference with multiple winmds handled? e.g., when the project has CppWinRTNamespaceMergeDepth set to > 1?  (some users require that for publishing specific namespace winmds)

Reply:
Do you have an example of this? I was definitely under the assumption that a single project outputs a single WinMD.

Reply:
That's true (and required) for Xaml projects, but we have customers that specifically require multiple winmds, not merged.  Easy enough to test with a couple nested namespaces under the project root namespace -> root.a.winmd, root.b.winmd -> an app should be able to project and consume classes from both